custom-trait-examples/custom-string-trait/README.md

```diff
@@ -29,7 +29,7 @@ Finally, update the `io.smithy.examples.traits.JsonNameTrait` entry in the
 `src/main/resources/META-INF/services/software.amazon.smithy.model.traits.TraitService` file to reflect the
 new name and package of your updated trait.
```

Comment:
lines 24-30 may need to be removed. similarly in the other READMEs for custom-annotation-trait and custom-structure-trait examples.

Reply:
Moreover, the parent [README](https://github.com/smithy-lang/smithy-examples/blob/main/custom-trait-examples/README.md) should probably also be updated, to talk about that these examples use the trait-package plugin that automaticaly takes care of the Java class generation and its discovery via META-INF.

Reply:
It also makes me wonder if there should be an example for hand written Java class for trait that's not covered by use cases supported by the trait-package plugin. Are there such use cases? If there are, we can add an example for that, though it might be uncommon, so we needn't block this PR on adding that use case.

Reply:
I can add a new example for a mix of hand-written and generated (you can do both in one package). 
I wonder if we could condense to have 2 examples: 
1. Only codegen with a variety of traits. 
2. Mixed codegen with hand written and not.

Reply:
Yeah, was thinking mix would also be a good example to show. Your plan above sounds good.